securedrop/manage.py

```diff
@@ -22,13 +22,9 @@
 # We need to import config in each function because we're running the tests
 # directly, so it's important to set the environment correctly, depending on
 # development or testing, before importing config.
-#
-# TODO: do we need to store *_PIDFILE in the application config? It seems like
-# an implementation detail that is specifc to this management script.
-
 os.environ['SECUREDROP_ENV'] = 'dev'
 
-WORKER_PIDFILE = "/tmp/test_rqworker.pid"
+from config import WORKER_PIDFILE
```

Comment:
You actually _don't_ want to do this, because we want to launch a separate rqworker instance just for the tests. It makes it easier for us to create a separate logfile just for the output from the tests, and avoids potential conflicts between test runs and the rqworker used on the development/staging machines.

It might be clearer to instead rename `WORKER_PIDFILE` to `TEST_WORKER_PIDFILE` throughout `manage.py`.